libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFSDKLoginViewControllerConfig.m

```diff
@@ -29,7 +29,13 @@
 #import <Foundation/Foundation.h>
 #import "SFSDKLoginViewControllerConfig.h"
 #import "UIColor+SFColors.h"
+#import "SFSDKAppConfig.h"
```

Comment:
We probably don't need this import anymore either.

Reply:
import is required. Since we access a property.